src/raftstore/store/peer.rs

```diff
@@ -796,6 +734,7 @@ impl Peer {
 
         // If applied index's term is differ from current raft's term, leader transfer
         // must happened, if read locally, we may read old value.
+        // TODO: use raft_log's applied_index to check instead.
```

Comment:
It should be fine to check term only, since the local read request is performed in sequence with apply action by apply worker.